plugins/azad-proxy.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: azad-proxy
+spec:
+  platforms:
+  - uri: https://github.com/XenitAB/azad-kube-proxy/releases/download/v0.0.3/kubectl-azad_proxy-darwin-amd64.zip
+    sha256: 51ae0098cbbb0b5da01f5b086c02474a8b74012e5f373c2dbeea36ad659c65e2
+    bin: kubectl-azad_proxy
+    files:
+    - from: kubectl-azad_proxy
+      to: .
+    - from: LICENSE
+      to: .
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  - uri: https://github.com/XenitAB/azad-kube-proxy/releases/download/v0.0.3/kubectl-azad_proxy-linux-amd64.zip
+    sha256: 323182ba7960e3b587c5d7b6971e0aaad29c0d5d1acb4d276e20c7f7c5c3ab98
+    bin: kubectl-azad_proxy
+    files:
+    - from: kubectl-azad_proxy
+      to: .
+    - from: LICENSE
+      to: .
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - uri: https://github.com/XenitAB/azad-kube-proxy/releases/download/v0.0.3/kubectl-azad_proxy-windows-amd64.zip
+    sha256: 5556aa4a6b679fe8e33639fd9df008290f2a3b9f0b965dc2fcdde0c913721233
+    bin: kubectl-azad_proxy.exe
+    files:
+    - from: kubectl-azad_proxy.exe
+      to: .
+    - from: LICENSE
+      to: .
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+  version: "v0.0.3"
+  shortDescription: Generate and handle authentication for azad-kube-proxy
+  description: |
+    Works together with the proxy (Azure AD Kubernetes API Proxy: azad-kube-proxy) to make it easier to generate
```

Comment:
This sounds like users need to have `azad-kube-proxy` installed. So please include a link where they can download this executable, and add a `caveat` section about this.

Reply:
Hi @corneliusweig!

This plugin is only used in combination with the proxy (and both are located in the GitHub link).

I don't see people using the plugin and then going for the proxy - wouldn't do much for them.

In this case, I am pointing the users of the proxy here to easily get the plugin.

Do you still want me to add more? (Since the GitHub link is already there)

Reply:
Caveat added! :+1: